tests/integrations/test_crossed_integrations.py

```diff
@@ -186,3 +244,94 @@ def validate_kafka_spans(self, producer_interface, consumer_interface, topic):
 
         # returns both span for any custom check
         return producer_span, consumer_span
+
+
+MISSING_LIBRARY_LOGIC = (
+    context.library == "cpp"
+    or context.library == "php"
+    or context.library == "dotnet"
+    or (context.library == "python" and context.weblog_variant != "flask-poc")
+    or (context.library == "golang" and context.weblog_variant != "net-http")
+    or (context.library == "java" and context.weblog_variant != "spring-boot")
+    or (context.library == "nodejs" and context.weblog_variant != "express4")
+    or (context.library == "ruby" and context.weblog_variant != "rails70")
+)
+
+
+@irrelevant(MISSING_LIBRARY_LOGIC)
+@scenarios.crossed_tracing_libraries
+@coverage.basic
+@features.kafkaspan_creationcontext_propagation_with_dd_trace_js
+class Test_NodeJSKafka(_Test_Kafka):
+    buddy_interface = interfaces.nodejs_buddy
+    buddy = _nodejs_buddy
+    WEBLOG_TO_BUDDY_TOPIC = f"Test_NodeJSKafka_weblog_to_buddy"
```

Comment:
```suggestion
    WEBLOG_TO_BUDDY_TOPIC = "Test_NodeJSKafka_weblog_to_buddy"
```

Same for other constants